docs/source/thanos.rst

```diff
@@ -0,0 +1,79 @@
+Using Thanos With Scylla Monitoring
+============================
+
+Scylla-Monitoring uses `Prometheus <https://prometheus.io/>`_ for metrics collection, which works out-of-the-box, but Prometheus does have limitations.
+`Thanos <https://thanos.io/>`_  is an opensource solution on top of Prometheus with multiple functionalities it can:
+
+1. Support high-availability.
+2. Support horizontal scaling.
+3. Support backup.
+
+A nice thing with Thanos approach is its flexibility, you can use some of its functionality depends on your need.
+
+Using Thanos As a Prometheus Aggregator
+----------------------------------------------------------
+There are a few reasons why you would need multiple Prometheus servers: if the total number of your time series reaches millions you can reach the limit of a single Prometheus server capacity.
+Sometimes it is also useful to limit the traffic between data centers, so you can have a Prometheus server per DC.
```

Comment:
```suggestion
In addtion, it may also be useful to limit the traffic between data centers. You can do this by having a dedicated Prometheus server per DC.
```